extensions/data-workspace/package.nls.json

```diff
@@ -7,8 +7,8 @@
 	"refresh-workspace-command": "Refresh",
 	"close-workspace-command": "Close Workspace",
 	"remove-project-command": "Remove Project",
-	"projects-view-no-workspace-content": "No workspace open, create new or open existing to get started.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about SQL Database projects [read our docs](https://aka.ms/azuredatastudio-sqlprojects).",
-	"projects-view-no-project-content": "No projects found in current workspace.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about SQL Database projects [read our docs](https://aka.ms/azuredatastudio-sqlprojects).\n",
+	"projects-view-no-workspace-content": "Create new or open existing to get started.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about projects [read our docs](https://aka.ms/azuredatastudio-projects).",
+	"projects-view-no-project-content": "No projects found in current workspace.\n[Create new](command:projects.new)\n[Open existing](command:projects.openExisting)\nTo learn more about projects [read our docs](https://aka.ms/azuredatastudio-projects).\n",
```

Comment:
do we still want to keep the "No projects found in current workspace" text when a workspace is open but no projects have been added to it yet?

Reply:
I suggest that the message in line 11 reads either "Create new or open existing to get started."  or "No projects open in current workspace, create new or open existing to get started." 
(+ buttons + docs link)

Reply:
I will update it to "No project open in current workspace, ....". 
Thanks!